packages/databyss-services/pages/PageProvider.js

```diff
@@ -9,8 +12,18 @@ export const PageContext = createContext()
 const PageProvider = ({ children, initialState }) => {
   const [state, dispatch, stateRef] = useReducer(reducer, initialState)
 
+  const fetchPages = () => {
```

Comment:
There would be no way of knowing in our storybook of what page to load since an `_id` is required to look up a page in the API. This is currently used for the initial mount of `Page` under our `Services` in storybook. The `EditorLoader` component is a consumer of `withPages`.